src/models/TalkMapper.php

```diff
@@ -784,4 +784,16 @@ public function getSpeakerFromTalk($talk_id, $display_name)
         return count($speakers) > 0 ? $speakers[0] : false;
 
     }
+
+    public function removeSpeakerFromTalk($talk_id, $speaker_id)
+    {
+        $params = [
+            'talk_id' => $talk_id,
+            'speaker_id' => $speaker_id,
+        ];
+
+        $sql = 'update talk_speaker set speaker_id = null where talk_id = :talk_id and speaker_id = :speaker_id';
```

Comment:
@heiglandreas - yes, that is already done - if you are a host, and you want to completely remove a speaker from a talk (not just unlink it), then that is done as part of editing a talk, which is already live.